tests/verification/test_opencodelists.py

```diff
@@ -0,0 +1,39 @@
+import pytest
+
+from jobserver.opencodelists import OpenCodelistsAPI
+
+from ..fakes import FakeOpenCodelistsAPI
+from .utils import compare
+
+
+pytestmark = [
+    pytest.mark.verification,
+]
+
+
+@pytest.fixture
+def opencodelists_api():
+    """create a new API instance for these tests"""
+    return OpenCodelistsAPI()
+
+
+def test_get_codelists(enable_network, opencodelists_api):
+    args = ["snomedct"]
+
+    real = opencodelists_api.get_codelists(*args)
+    fake = FakeOpenCodelistsAPI().get_codelists(*args)
+
+    compare(fake, real)
+
+    assert real is not None
+
+
+def test_get_codelists_with_unknown_coding_system(enable_network, opencodelists_api):
+    args = ["test"]
+
+    real = opencodelists_api.get_codelists(*args)
+    fake = FakeOpenCodelistsAPI().get_codelists(*args)
+
+    compare(fake, real)
```

Comment:
I'm not sure what the comparison with the fake api is adding here, given that we're checking below that the expected response is always an empty list?

Reply:
It compares the two results without me having to think too hard about what they are!  I could reformat the test to directly check the response from each api object is `[]`, but having done this for lots of methods in the GitHub tests I found that consistency was much easier to read.